src/lib/api/builder.ts

```diff
@@ -558,7 +558,7 @@ export class BuilderAPI extends BaseAPI {
         headers = { ...config.headers }
       }
     }
-    const authHeaders = await this.authorization.createAuthHeaders(method, path)
+    const authHeaders = this.authorization.createAuthHeaders(method, path)
```

Comment:
is the method `createAuthHeaders` no longer async?

Reply:
No longer async :D